src/components/30-organisms/footer/index.js

```diff
@@ -0,0 +1,284 @@
+// TODO fix that stuff
+/* eslint-disable import/no-extraneous-dependencies */
+import { CaretSvg } from '@axa-ch/materials/icons';
+import '@axa-ch/container';
+import { LitElement, html, svg, css, unsafeCSS } from 'lit-element';
+import defineOnce from '../../../utils/define-once';
+import styles from './index.scss';
+
+const _listElementHasNoContent = label => {
+  return !label || label.toString() === '[object Text]';
+};
+
+const _extractNestedHref = ev => {
+  const { target } = ev;
+  let element;
+  if (!target || !target.href) {
+    element = ev;
+    while (!element.href) {
+      if (element.target && element.target.parentNode) {
+        element = element.target.parentNode;
+      } else if (element.parentNode) {
+        element = element.parentNode;
+      } else {
+        return undefined;
+      }
+    }
+  }
+  return element === undefined ? target.href : element.href;
+};
+
+class AXAFooter extends LitElement {
+  static get tagName() {
+    return 'axa-footer';
+  }
+
+  static get styles() {
+    return css`
+      ${unsafeCSS(styles)}
+    `;
+  }
+
+  static get properties() {
+    return {
+      // 'clickevents' prevents the throwing of native click events and sends a custom axa-footer-click event.
+      clickevents: { type: Boolean },
+    };
+  }
+
+  constructor() {
+    super();
+    this.clickevents = false;
+    this.onItemClick = () => {};
+    this._accordionActiveIndex = -1;
+  }
+
+  render() {
+    return html`
+      <footer class="o-footer">
+        <axa-container>
+          <div class="o-footer__content">
+            <div class="o-footer__collection">
+              <div class="o-footer__main">
+                <slot
+                  name="column-0-title-desktop"
+                  class="o-footer__title-desktop"
+                ></slot>
+                <button
+                  class="o-footer__accordion-button"
+                  @click="${() => this._handleAccordionClick(0)}"
```

Comment:
What's the '0' for in the function argument?

Reply:
The index of the accordion button.